src/views/VoucherView/index.tsx

```diff
@@ -61,7 +61,7 @@ export const VoucherView = () => {
         </Grid>
 
         <Grid item md={4} lg={3}>
-          <SectionUpcomingEvents widthValue="100%" />
+          <SectionUpcomingEvents widthValue="100%" upcomingEvents={[]} />
```

Comment:
```suggestion
          <SectionUpcomingEvents widthValue="100%" upcomingEvents={[]} />
```
ini minta tolong get events benerannya yakss

Reply:
okeeh